itest/rust/src/object_tests/onready_test.rs

```diff
@@ -267,6 +284,44 @@ impl INode for OnReadyWithImplWithoutReady {
 
 // ----------------------------------------------------------------------------------------------------------------------------------------------
 
+// Class that has overriden ready with Gd<Self> receiver. Used to test whether variables are still initialized.
+#[derive(GodotClass)]
+#[class(no_init, base=Node)]
+struct OnReadyWithImplGdSelf {
+    base: Base<Node>,
+    #[hint(onready)]
+    auto: OnReady<i32>,
+    #[hint(no_onready)]
+    manual: OnReady<i32>,
+    runs_manual_init: bool,
+}
+impl OnReadyWithImplGdSelf {
+    fn create(runs_manual_init: bool) -> Gd<OnReadyWithImplGdSelf> {
+        Gd::from_init_fn(|base| Self {
```

Comment:
Nitpick: multi-line item definitions should always have an space between each other...
```suggestion
    runs_manual_init: bool,
}

impl OnReadyWithImplGdSelf {
    fn create(runs_manual_init: bool) -> Gd<OnReadyWithImplGdSelf> {
        Gd::from_init_fn(|base| Self {
```

Same above for `#[itest]`